FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -2196,6 +2196,14 @@ - (BOOL)useUserAccessGroup:(NSString *_Nullable)accessGroup
   return YES;
 }
 
+- (BOOL)useUserAccessGroup:(NSString *_Nullable)accessGroup
+                     error:(NSError *_Nullable *_Nullable)outError {
+  // self.storedUserManager is initialized asynchronously. Make sure it is done.
+  dispatch_sync(FIRAuthGlobalWorkQueue(), ^{
```

Comment:
I'm thinking maybe it makes sense to load storedUserManager synchronously, but it requires more investigation on the impact. This should be good for now.